src/Symfony/Component/Console/Helper/Table.php

```diff
@@ -270,6 +298,14 @@ public function addRow($row)
         return $this;
     }
 
+    /**
+     * Sets the row.
+     *
+     * @param int   $column
+     * @param array $row
+     *
+     * @return Table
+     */
     public function setRow($column, array $row)
```

Comment:
I would only keep the `@return` here. Everything else does not seem adding value.